.github/workflows/prettier.yml

```diff
@@ -0,0 +1,53 @@
+name: Prettier
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches:
+      - main
+      - ann-kilzer-patch-1
```

Comment:
should we only have main branch?

Reply:
No, I need to test it on this very branch